lib/tmi/client.ex

```diff
@@ -84,7 +84,9 @@ defmodule TMI.Client do
   """
   @spec connect_ssl(Conn.t()) :: :ok | {:error, any()}
   def connect_ssl(%Conn{} = conn) do
-    Client.connect_ssl!(conn.client, conn.server, conn.port)
+    options = [verify: :verify_peer, cacerts: :public_key.cacerts_get()]
```

Comment:
This fails at runtime if no certs could be loaded: https://github.com/erlang/otp/blob/99e81e261096d9390fa2691d9a6f98589848c200/lib/public_key/src/pubkey_os_cacerts.erl#L42-L46

So I suppose there could be a rescue wrapper here that defaults to `options = [verify: :verify_none]` in that case.